static/template.palette

```diff
@@ -825,6 +825,32 @@
             "inputPorts": [],
             "readonly": true
         },
+        {
+            "category": "EnvironmentVars",
```

Comment:
Sorry, I think I changed this to "EnvironmentVariables" in Eagle.ts

That might have been the cause of the conflicts with the master branch.